packages/create-rise/src/cli.ts

```diff
@@ -73,9 +62,62 @@ async function createRise(opts: Options) {
     await fs.mkdir(root)
   }
 
-  await downloadAndExtractTemplate(root, template)
+  const template = await select({
+    message: 'Choose a template',
+    choices: [
+      {
+        value: 'blank',
+        name: 'I want to start from scratch',
+      },
+      {
+        value: 'full',
+        name: 'I want to start with all the examples',
+      },
+      {
+        value: 'custom',
+        name: 'I will choose the example myself',
+      },
+    ] as const,
+    theme: prompt,
+  })
+
+  await downloadAndExtractTemplate(
+    root,
+    template === 'blank' ? '@rise-tools/template-blank' : '@rise-tools/template-full'
+  )
   await copyAdditionalTemplateFiles(root)
 
+  const templateRoot = path.join(root, 'src')
+
+  const entryPointPath = path.join(templateRoot, 'server.ts')
+  const entryPoint = await fs.readFile(entryPointPath, 'utf-8')
+
+  // Replace the import with the selected example
+  if (template === 'custom') {
+    const example = await select({
+      message: 'Choose the example you want to start with:',
+      choices: AVAILABLE_EXAMPLE_CHOICES,
+    })
+
+    const toRemove = [
+      'home',
+      ...AVAILABLE_EXAMPLE_CHOICES.map((e) => e.value).filter((name) => name !== example),
+    ]
+    for (const folder of toRemove) {
+      await fs.rm(path.join(templateRoot, folder), { force: true, recursive: true })
+    }
+
+    await fs.writeFile(
+      entryPointPath,
+      entryPoint.replace(MODELS_IMPORT_REGEX, `import { models } from './${example}/ui'`)
```

Comment:
Lol this is so hacky, but I'm not sure what else I would do to keep things somewhat clean

One obvious issue is that we should assign the appropriate example as the entry path `""`

Maybe we can find a cleaner way to do this, I'll think about it but lets chat more on discord

Reply:
Yeah, this is good old technique that we used back when we had first templates in React Native. It works and does not require any external library. This is also rather isolated use case and unless we plan to have more of this in the future, I think it's fine to have it done this way!